src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/CodeReader.cs

```diff
@@ -74,8 +74,8 @@ public class CodeReader
 
         private static void PostProcessWebUris(ProjectAuthenticationSettings projectAuthenticationSettings)
         {
-            bool isBlazorWasm = projectAuthenticationSettings.ApplicationParameters.IsBlazorWasm
-                && !projectAuthenticationSettings.ApplicationParameters.IsWebApp;
+            bool isBlazorWasm = projectAuthenticationSettings.ApplicationParameters.IsBlazorWasm.HasValue && projectAuthenticationSettings.ApplicationParameters.IsBlazorWasm.Value
```

Comment:
(nit) May as well move each expression (separated by &&) onto its own line since they are long and equal in priority.

Reply:
sure yeah.